Web/Edubase.Web.UI/Web.config

```diff
@@ -285,8 +285,11 @@
     <add key="SfbClient_Timeout" value="2" />
     <add key="OSPlacesApiServices_RetryIntervals" value="1,2,2,4"/>
     <add key="AzureMapService_RetryIntervals" value="1,2,2,4"/>
-    
 
+    <add key="AzureMapServiceTimeoutKey" value="2"/>
+    <add key="OSPlacesApiServicesTimeoutKey" value="10"/>
```

Comment:
What is the rationale for these being named differently to and being added separately to the `xxx_Timeout` timeout values above please?

Reply:
To identify these values differently from the other timeouts, since these timeouts also had retries - I will move them back with the others and rename them, as you can also compare them to the retry list (if there is a lot more added it might be a good idea to identify them this way?)